scripts/build-core.sh

```diff
@@ -1,12 +1,14 @@
 #!/bin/sh
+set -e
+
 export PATH="/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin"
 export PATH="$HOME/.cargo/bin:$PATH"
 
 FILENAME="libdeltachat.a"
 DIR="../deltachat-ios/libraries"
 
 # Delete old build, if any.
-rm -f "${DIR}/${FILENAME}"
+rm -f "${DIR}/${FILENAME}" || true
```

Comment:
```suggestion
rm -f "${DIR}/${FILENAME}"
```
because `-f` already ignores the file does not exist error